recipes-devtools/clang/clang/0022-compiler-rt-Use-mcr-based-barrier-on-armv6.patch

```diff
@@ -63,7 +63,7 @@ index 69a3d8620f92..60bac86df218 100644
 +#elif __ARM_ARCH >= 6
 +#define DMB mcr p15, #0, r0, c7, c10, #5
 +#else
-+#error Only use this with ARMv6+
++#define DMB asm volatile ("" : : : "memory")
```

Comment:
Can you try backporting https://github.com/llvm/llvm-project/commit/1c9415806ba6d0d48a160637eea7d1b70efaae69 instead ?